cmd/gator/test/test.go

```diff
@@ -117,10 +150,32 @@ func formatOutput(flagOutput string, results []*test.GatorResult) string {
 			errFatalf("pre-unmarshaling results from json: %v", err)
 		}
 
-		yamlb, err := yaml.Marshal(unmarshalled)
-		if err != nil {
-			errFatalf("marshaling validation yaml results: %v", err)
+		var yamlb []byte
+		if stats != nil {
+			statsAndResuluts := map[string]interface{}{"results": results, "stats": stats}
```

Comment:
This appears to ignore the `GetYamlFriendlyResults` logic above.

Reply:
Oh yes, I will use `yamlResults` instead of `results`